icu4j/readme.html

```diff
@@ -876,7 +876,7 @@ <h5> Generating Data from CLDR </h5>
 <I> Note: This procedure assumes that all 3 sources are present</I>
 <ol>
     <li>Checkout or download CLDR version 'release-38'</li>
```

Comment:
We should probably update the CLDR tag to 'release-39' as well.

Reply:
Ah thanks for this :)